.github/workflows/auto-create-release-pr.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     outputs:
       semver: ${{ steps.out.outputs.semver }}
       previous_ref: ${{ steps.out.outputs.previous_ref }}
+      proceed: ${{ steps.out.outputs.proceed }}
```

Comment:
All changes in this file come from this PR https://github.com/MetaMask/metamask-extension/pull/35662, which was not included in 13.3.0, but was cherry-picked in 13.2.1.